### DISCUSSION TITLE
Question: Sunshine / Moonlight suddenly not blurry anymore with hevc since last Sunshine nightly

### DISCUSSION THREAD

**Discussion by lastmetroid** [LizardByte/community#599] [Sunshine]
I have a general question:

Host: RTX4080 Windows 11 24h2
Client: Intel UHD630 and N100 Windows 11 24h2, Moonlight

I had the previous problem since years with Moonlight / Sunshine on a Intel UHD 630 client with hevc or h264, that the picture was somehow blurry, especially noticable on text/font or 1 pixel wide lines like Icons, borders and so on.

I bought a new mini Pc with Intel N100 which supports AV1 decoding, so I used that for a while and I noticed, when I selected AV1 and experimental YUV 4:2:2 in Moonlight, the blurry issue was gone.

That option doesnt work though on Intel UHD630.

I installed latest Sunshine nightly (v2024.1217.154237) today, and for some reason, the blurry issue is now alo fixed on my Intel UHD630 with hevc and YUV 4:2:2 off. I had used nightly before mostly from a few days ago, and there it was still blurry.

Why? What was changed? Please keep this change!

**Comment by ReenigneArcher** [comment]
Probably fixed upstream by ffmpeg.

[a comment by an automated account is not shown]
